explorer/src/components/DetailTable.tsx

```diff
@@ -37,9 +37,20 @@ function formatCellValues(val: string | number, field: string) {
       </Box>
     );
   }
+
   if (field === 'bond') {
     return unymToNym(val, 6);
   }
+
+  if (field === 'owner') {
+    console.log(val);
```

Comment:
Did you forget maybe this log @fmtabbara ?

Reply:
I did. Thanks!